terra2/assetlist.json

```diff
@@ -11,7 +11,7 @@
         },
         {
           "denom": "luna",
-          "exponent": 6
+          "exponent": 18
```

Comment:
I'm skeptical about this change to 18 and want to say it's wrong... Are you certain LUNA has exponent 18?

Reply:
You are absolutely right. I meant to change line 525. The exponent of wstETH should be changed to 18. I made a mistake and initially put it at 6. When I went back to edit, I changed the wrong line.

Reply:
Ok, please make the necessary corrections

Reply:
Corrections made.